script.js

```diff
@@ -1,79 +1,79 @@
 /* eslint-disable no-use-before-define */
 
-let books = [];
+class Store {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    Store.books.push(this);
+  }
+
+  addBook(book) {
+    this.books.push(book);
+    displayBook();
+    Stringifier();
+  }
+
+  bookRemover(index) {
+    this.books.splice(index, 1);
+    displayBook();
+    Stringifier();
+  }
+
+  static books = [];
+}
+
+const store = new Store();
 
 // Update books object from local storage
 
 let oldBooks = localStorage.getItem('book');
 if (oldBooks !== null) {
   oldBooks = JSON.parse(oldBooks);
-  books = oldBooks;
+  store.books = oldBooks;
 }
 
+// Function to dynamically render the books
+
 const title = document.querySelector('#title');
 const author = document.querySelector('#author');
 const form = document.querySelector('#form');
 
-// Function to remove childnodes from the book container
-
-function removeChild() {
+function displayBook() {
   const bookHolder = document.querySelector('.book-holder');
   while (bookHolder.hasChildNodes()) {
     bookHolder.removeChild(bookHolder.firstChild);
   }
-}
-
-// Function to dynamically render the books
-
-function displayBook() {
-  const bookHolder = document.querySelector('.book-holder');
-  books.map((book, index) => {
-    const singleBook = document.createElement('div');
-    singleBook.className = book.title;
-    const title = document.createElement('p');
-    title.innerHTML = book.title;
-    const author = document.createElement('p');
-    author.innerHTML = book.author;
+  store.books.map((book, index) => {
+    const singleBook = document.createElement('tr');
+    const title = document.createElement('td');
+    title.innerText = `"${book.title}" by ${book.author}`;
+    const button = document.createElement('td');
```

Comment:
- [ ] When I'm trying to add a book to the collection, getting the below error. Please fix this issue and make sure that users should be able to add and remove books successfully.
![image](https://user-images.githubusercontent.com/83803414/197854876-10143bb3-a16f-40bb-9596-ced9cf955245.png)

Reply:
Okay thanks @thecodechaser for the feedback, we have addressed the issue!🙏 It was giving that because we used static for the "books" array inside the class, which was inaccessible by the instance. Thanks again!